crates/sui/src/client_ptb/ptb.rs

```diff
@@ -157,6 +164,17 @@ impl PTB {
             program_metadata.gas_budget.value,
             gas_price,
         );
+
+        if program_metadata.serialize_unsigned_set {
+            serialize_or_execute!(tx_data, true, false, context, PTB).print(true);
```

Comment:
Can we use `serialize_or_execute` to do more of what's happening below? (Execute the transaction and display it?) It seems like that would help with consistency in general.

Reply:
I think we can do that once we add the `summary` flag to other commands. Then I can probably have everything within this macro for PTB.